io.go

```diff
@@ -89,6 +90,32 @@ func NewIOWithTerminal(stdin io.Reader, stdout, stderr io.Writer, terminal bool)
 	}
 }
 
+//NewIOWithFifoSet returns IOCreation handler function for a custom fifo path
+func NewIOWithFifoSet(stdin io.Reader, stdout, stderr io.Writer, paths *FIFOSet, terminal bool) IOCreation {
```

Comment:
Mentioned [here](https://github.com/kubernetes-incubator/cri-containerd/pull/113#discussion_r132358401).

We may not need this function, because we are fine that FIFOs are created by the containerd client at specified place `/tmp`.
The only thing we could not do with current `NewIO` is that we could not disable stdin. There is no way to pass empty stdin fifo path to current containerd client.

To solve this problem, we could:
* Option 1: Create fifos ourselves, and pass them to containerd client. (What this function is doing)
* Option 2: Still use containerd.NewIO, but add an option to disable stdin.
* Option 3: Use containerd.NewIO directly, and call task.CloseIO afterwards.

Seems that option 2 is better.